CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+#### Added
+- Added another template with option of a configurable message
```

Comment:
add an example of this auth proc filter configuration to the README (with all options)